lib/config/executor/network_interface.go

```diff
@@ -66,4 +67,6 @@ type ContainerNetwork struct {
 
 	// set of network wide links and aliases for this container on this network
 	Aliases []string `vic:"0.1" scope:"hidden" key:"aliases"`
+
+	PortGroup object.NetworkReference `vic:"0.1" scope:"read-only" recurse:"depth=0"`
```

Comment:
this is a runtime variable, but as it's referenced in the map object. It's not easy to address that issue without hook that in here.